tools/functional-tester/etcd-tester/tester.go

```diff
@@ -110,7 +110,8 @@ func (tt *tester) runLoop() {
 				}
 			}
 			if !ok {
-				log.Printf("etcd-tester: [round#%d case#%d] checking current revisions failure...", i, j)
+				time.Sleep(time.Second)
```

Comment:
who names the directory by timestamp? can we resolve at agent side? if it cannot rename due to a naming conflict, it can just sleep and wait.